unity-sdk/AndroidPlugin/settings.gradle

```diff
@@ -15,3 +15,4 @@ dependencyResolutionManagement {
 rootProject.name = "unity"
 include ':app'
 include ':OptimoveUnityPlugin'
+include ':UnityActivity'
```

Comment:
here is a SO question (which mb you saw) https://stackoverflow.com/questions/61865739/cannot-find-the-unityplayeractivity-class-inside-com-unity3d-player

so, this is prob fine. They all suggest copying unity activity one way or another

Reply:
i can come up with another way via gradle project manipulation (same way as we add google-services.json)
1. copy our overriding activity into streaming assets
2. before gradle is building copy into UnityPlugin module

but perhaps even more hacky

Reply:
If there is another plugin doing same (extending UnityActivity), consumers could somehow merge all these extensions

Reply:
I will add to the docs a section that explains why we have added UnityActivity.aar